### ISSUE TITLE
Finer grain user permission

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#377]
```
E.G.
Allow permissions to download. (Maybe include rating filter in download option).
Update server permissions.
Run import
ETC
```

Original issue reported on code.google.com by `Craige1Emmott` on 14 Jan 2013 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
I don't see a need for this? Including a rating filer in the download option 
would be difficult. The only rating filter you could possibly put in place is 
no adult material allowed. There is no centralized repository for ratings apart 
from anidb which only has  Adult or not adult.


```

Original comment by `imperia...@gmail.com` on 15 Jan 2013 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
The main use of this is to allow none admin users to download. I don't see why 
a non admin user shouldn't be allowed to download, but I do see some people not 
allowing users from downloading, for reasons like data usage concerns. The 
permissions allow for both use cases. 

Ratings would be the "hide category's" filter already in the users settings. 
The main step that would need to be completed before this can work is pulling 
the info from the torrent search results, which is in another issue ticket. 
Once the anime can be established from the torrent result, the categories can 
be pulled from anidb.net and then the hide category filter would be applied.


```

Original comment by `Craige1Emmott` on 16 Jan 2013 at 11:11

**Comment by GoogleCodeExporter** [comment]
```
Agree with better permissions.

Although doing it via rating is impossible.
The anime data shown in the download areas, is simply a guess based on the 
torrent names. 

I do see where you are going with this, but I don't think the scraping would 
ever be reliable enough to base rules on.

```

Original comment by `werndly...@gmail.com` on 21 Jan 2013 at 12:28
- Changed state: **Accepted**
- Added labels: **Product-JMM**

**Comment by GoogleCodeExporter** [comment]
```
Give me some time to work on getting the scrapper to be more reliable for use 
on the downloading rating side. The other options like run import and update 
server permissions should be "fairly easy".
```

Original comment by `Craige1Emmott` on 21 Jan 2013 at 10:47

**Comment by GoogleCodeExporter** [comment]
```
https://github.com/japanesemediamanager/jmmclient/issues/36
```

Original comment by `werndly...@gmail.com` on 19 Aug 2014 at 7:04
- Changed state: **Migrated**
